BUILD.bazel

```diff
@@ -346,6 +347,15 @@ container_push(
     tag = "$(container_tag)",
 )
 
+container_push(
+    name = "push-slirp-hook-sidecar",
```

Comment:
Have you requested to the quay.io container to be created?

Reply:
Per my understanding having the image built as part of cluster-sync will also include in the release, is that right?

Reply:
```
+container_push(
+    name = "push-network-slirp-binding",
+    format = "Docker",
+    image = "//cmd/network-slirp-binding:network-slirp-binding-image",
+    registry = "$(container_prefix)",
+    repository = "$(image_prefix)network-slirp-binding",
+    tag = "$(container_tag)",
+)
```

I meant that there is no `network-slirp-binding` repository in `kubevirt/network-slirp-binding` like we have for [smbios sidecar example](https://quay.io/repository/kubevirt/example-hook-sidecar). If you want to create it, you should ping @dhiller and the nightly builds will update the repository.